docs/wrappers/ledgerutil_postscript.md

```diff
@@ -29,4 +33,19 @@ Here is an example of the `ledgerutil compare` command.
 
   * Note that both snapshot locations must be accessible by the command, for example by mounting volumes from two different peers, or by copying the snapshots to a common location.
 
+### ledgerutil identifytxs example
+
+Here is an example of the `ledgerutil identifytxs` command.
+
+  * Identify relevant transactions in mychannel from a block store within an allowed block range.
```

Comment:
Expand on what "allowed block range means". At least mention the input JSON file.